run.sh

```diff
@@ -6,7 +6,7 @@ SRC_DIR="/work/ITK"
 BLD_DIR="/work/ITK-bld"
 
 # If the SRC_DIR is not mounted, then this tag will be checked out
-GIT_TAG=${GIT_TAG:-"master"}
+GIT_TAG=${GIT_TAG:-"main"}
```

Comment:
@thewtex  the main ITK repo has not been renamed yet? or is there going to be an alias available soon?

Reply:
@blowekamp good catch -- please take a look at the corresponding follow-up in #16